src/components/calcite-input-time-picker/calcite-input-time-picker.tsx

```diff
@@ -89,6 +80,14 @@ export class CalciteInputTimePicker implements LabelableComponent {
   /** aria-label for the second up button */
   @Prop() intlSecondUp?: string;
 
+  /** BCP 47 language tag for desired language and country format */
+  @Prop() lang: string = document.documentElement.lang || navigator.language;
```

Comment:
I don't think we should set this prop by default. We can have a `@State` variable that looks up the lang if this isn't set.

Reply:
Agreed.  This gives a Stencil compile warning anyway because its already defined on the prototype.  My approach I think at this point is to convert `locale` into a State variable that syncs with the `lang` attribute.